src/components/Reports/TeamReport/TeamReport.jsx

```diff
@@ -320,6 +345,7 @@ export function TeamReport({ match }) {
           <div className="input-group input-group-sm d-flex flex-nowrap justify-content-between active-inactive-container">
             <div className="d-flex align-items-center">
               <div className="d-flex flex-column">
+                {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```

Comment:
If you try your best and still cannot fix it, we may want to set this as eslint rule, let me know your thoughts.